index.html

```diff
@@ -3032,7 +3042,7 @@ <h1>Artists</h1>
             <p id="statM13" alt="statM13" label="statM13" class="statPortion statTotal">TOTAL FREE MULTIPLIERS: <span id="sM13" alt="sM13" label="sM13" class="statNumber statTotal">0</span></p>
         </div>
         <div id="offeringMultiplierStats" alt="offeringMultiplierStats" label="offeringMultiplierStats" class="statContainer" style="display: none;">
-            <p id="offeringStatTitle" alt="offeringStatTitle" label="offeringStatTitle" class='statPortion' style="color: orangered; font-size: 1.2em">Offering Multipliers</p>
+            <p id="offeringStatTitle" alt="offeringStatTitle" label="offeringStatTitle" class="statPortion" style="font-size: 1.2em">Offering Multipliers</p>
```

Comment:
was the color removed here purposefully?

Reply:
Moved into CSS